ui/cypress/e2e/onboarding.test.ts

```diff
@@ -0,0 +1,359 @@
+interface TestUser {
+  username: string
+  password: string
+  org: string
+  bucket: string
+}
+
+const defTimeOut = 3000
+
+describe('Onboarding', () => {
+  let user: TestUser
+
+  beforeEach(() => {
+    cy.flush()
+
+    cy.fixture('user').then(u => {
+      user = u
+    })
+
+    cy.visit('/')
+  })
+
+  it('Can Onboard to Quick Start', () => {
+    //Check splash page
+    cy.location('pathname', {timeout: defTimeOut}).should(
+      'include',
+      'onboarding/0'
+    )
+    cy.get('h3.wizard-step--title').contains('Welcome to InfluxDB 2.0')
+    cy.get('div.wizard--credits').contains('Powered by')
+    cy.get('div.wizard--credits').contains('InfluxData')
+
+    //Continue
+    cy.get("button[title='Get Started']").click()
+
+    cy.location('pathname', {timeout: defTimeOut}).should(
+      'include',
+      'onboarding/1'
+    )
+    //Check navigation bar
+    cy.get("div.wizard--progress-title.checkmark:contains('Welcome')").click()
+
+    //Check splash page
+    cy.get('h3.wizard-step--title').contains('Welcome to InfluxDB 2.0')
+    cy.get('div.wizard--credits').contains('Powered by')
+    cy.get('div.wizard--credits').contains('InfluxData')
```

Comment:
Assertions on specific copy are highly brittle and subject to change.

Reply:
replaced with getByTestID